changelogs/server_server/newsfragments/1623.clarification

```diff
@@ -0,0 +1 @@
+I have updated the list of Resolving server names in step 3 from unordered list to ordered list. Because we care about its order
```

Comment:
This style isn't consistent with our regular changelog entries. I suggest you have a look at the other entries in https://spec.matrix.org/v1.7/changelog/ and try to follow their style.